fabfile.py

```diff
@@ -183,6 +183,50 @@ def update_vendor_config():
         system("cd %s ; git pull" % config_file_dir)
 
 
+@task
+def docker_fabric_task():
+    """
+    To be launched with fab -c configs/bluenove-server-configs/dev_staging.rc docker_fabric_task
+    """
+    # env.docker_api_version = '1.17.05'
+    env.docker_tunnel_remote_port = 22024
+    from dockerfabric import cli
+    from dockerfabric.utils.files import temp_dir
+    cwd = os.getcwd()
+    if env.docker_piwik:
+        with temp_dir() as remote_tmp:
+            print remote_tmp
+            try:
+                run("git clone git@github.com:siutin/piwik-docker-compose.git")
+                run("cd piwik-docker-compose && docker-compose up -d")
+                run("docker exec -d piwikdockercompose_db_1 mysqldump --result-file=dump.sql")
+            except:
+                run("cd piwik-docker-compose && docker-compose up -d")
+                run("docker exec -d piwikdockercompose_db_1 mysqldump --result-file=dump.sql")
+        cli.copy_resource(u"piwikdockercompose_db_1", '/dump.sql', cwd + "/dump.sql")
+    else:
+        get('/etc/nginx/sites-available', cwd)
```

Comment:
I'm not sure I understand this line. What is it meant to do?

Reply:
@swergas this line download the data base backup in the case of not using piwik inside a docker container, and stores it on your localhost, normally, the databse backup is supposed to be in this location